portfolio.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+ <link href="https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Readex+Pro:wght@200&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="css/portfolio.css">
+  <title>Program Portfolio</title>
+</head>
+<body>
+  <header>
+   <center><h1>Programming Portfolio</h1></center>
```

Comment:
The center tag shouldn't be used anymore. We should use style rules (such as text-align) to achieve centered text.